src/CISDSC/Examples/CIS_Microsoft_Windows_10_Enterprise_Release_2004_With_LAPS.ps1

```diff
@@ -0,0 +1,45 @@
+Configuration Win10_2004_L1_With_LAPS
+{
+    Import-DSCResource -ModuleName 'PSDesiredStateConfiguration'
+    Import-DSCResource -ModuleName 'CISDSC' -Name 'CIS_Microsoft_Windows_10_Enterprise_Release_2004'
+
+    node 'localhost'
+    {
+        Package 'InstallLAPS' {
+            Name  = 'Local Administrator Password Solution'
+            #Path can be substituted for a file path in your envrionment
+            Path = 'https://download.microsoft.com/download/C/7/A/C7AAD914-A8A6-4904-88A1-29E657445D03/LAPS.x64.msi'
+            ProductId = 'EA8CB806-C109-4700-96B4-F1F268E5036C'
+        }
+
+        CIS_Microsoft_Windows_10_Enterprise_Release_2004 'CIS Benchmarks' {
+            #These exclusions are services that are not in default installs of Windows. Remove the exlusions if they are applicable to your envrionment.
```

Comment:
Same comment on this comment - can we move it into a block where a Get-Help call will find it?

Reply:
We will likely tackle this in #98.